do.py

```diff
@@ -1178,6 +1182,54 @@ def authors(ctx_obj, revision_args):
     except subprocess.CalledProcessError:
         print('Error caught: Incorrect revision start or revision end')
 
+"""
+- Currently the run-job/reuse fails (issue https://github.com/scipy/scipy/issues/16009)
+"""
+@cli.cls_cmd('act')
```

Comment:
I am not sure about the name `act`. Is it clear for new users? Maybe we should name this `ci` or something else.

Reply:
`ci` might seem a little generic+confusing. I was thinking about `act_ci`, does this sound better?

Reply:
My argument here is that `act` is not something newcomers know about whereas CI talks to anyone. And here it is an interface to play with what would be run in the CI. Let see what other think.

Reply:
Agreed, we want a generic name - we also use `test`, `build`, etc.

This will _only_ run GitHub Actions jobs _locally_, so how about `run-gh-action`? Or `ci-local`?

Reply:
+1 for `ci-local`

Reply:
`run-gh-action` does sound more intuitive to me.